nautobot_golden_config/tables.py

```diff
@@ -227,27 +193,62 @@ class GoldenConfigTable(BaseTable):
     """Table to display Config Management Status."""
 
     pk = ToggleColumn()
-    device__name = TemplateColumn(
-        template_code="""<a href="{% url 'dcim:device' pk=record.device.pk %}">{{ record.device }}</a>""",
+    name = TemplateColumn(
+        template_code="""<a href="{% url 'dcim:device' pk=record.pk %}">{{ record.name }}</a>""",
         verbose_name="Device",
     )
+
     if ENABLE_BACKUP:
-        backup_last_success_date = TemplateColumn(verbose_name="Backup Status", template_code=BACKUP_SUCCESS)
+        backup_last_success_date = Column(verbose_name="Backup Status", empty_values=())
     if ENABLE_INTENDED:
-        intended_last_success_date = TemplateColumn(verbose_name="Intended Status", template_code=INTENDED_SUCCESS)
+        intended_last_success_date = Column(verbose_name="Intended Status", empty_values=())
     if ENABLE_COMPLIANCE:
-        compliance_last_success_date = TemplateColumn(
-            verbose_name="Compliance Status", template_code=COMPLIANCE_SUCCESS
-        )
+        compliance_last_success_date = Column(verbose_name="Compliance Status", empty_values=())
 
     actions = TemplateColumn(
         template_code=ALL_ACTIONS, verbose_name="Actions", extra_context=CONFIG_FEATURES, orderable=False
     )
 
+    def render_backup_last_success_date(self, record, column):  # pylint: disable=no-self-use
+        """Pull back backup last success per row record."""
+        entry = record.goldenconfig_set.first()
+        if not hasattr(entry, "backup_last_success_date") or not hasattr(entry, "backup_last_attempt_date"):
+            return ""
```

Comment:
I think we return a `--` currently, can you confirm and update appropriately?

Reply:
confirmed -- used in demo instance.  Updated.